ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -373,8 +373,7 @@ objects:
             exit 1
         fi
 
-        /bin/openshift-install --dir=/tmp/artifacts/installer create cluster &
-        wait "$!"
+        /bin/openshift-install --dir=/tmp/artifacts/installer create cluster
```

Comment:
Why this change?

Reply:
That was an artifact of some previous refactoring. Backgrounding the process and then immediately waiting on it should be equivalent to just running it in the foreground.

Reply:
I've filed #2680, rolling this back and adding an additional `wait` to our `TERM` handlers.  As discussed in that PR, we need the backgrounded `create cluster` so the shell will fire the `TERM` handler which `TERM`s the child processes.  As master stands (with this PR but without #2680), aborting an e2e job will `TERM` the setup container's shell process, but that process will not forward the `TERM` to `openshift-install` and will wait until it exits normally.  I haven't actually tested with a wrapping container, but my guess is that this means that jobs being aborted during `openshift-install` are mostly hitting their gentle-shutdown timeout and getting `KILL`ed.  With #2680, the forwarded `TERM` gives the installer a chance at a gentle shutdown (although at the moment, I don't think the installer takes advantage of that opportunity).